docs/readme/train.md

```diff
@@ -47,109 +47,122 @@ maxatac train --genome hg38 --arch DCNN_V2 --sequence hg38.2bit --meta_file CTCF
 
 ### `--train_roi`
 
-This argument is used to input the bed file that you want to use to define the training regions of interest. If you set this option you will randomly select regions from this file for training instead of using the meta data to build the training data pool. 
+This argument is used to input the BED file that you want to use to define the training regions of interest. If you set this option, you will randomly select regions from this file for training instead of using the meta data to build the training data pool. 
 
 ### `--validate_roi`
 
-This argument is used to input the bed file that you want to use to define the validation regions of interest. If you set this option you will randomly select regions from this file for validation instead of using the meta data to build the validation data pool.
+This argument is used to input the BED file that you want to use to define the validation regions of interest. If you set this option, you will randomly select regions from this file for validation instead of using the meta data to build the validation data pool.
 
 ### `--target_scale_factor`
 
 The scaling factor for scaling model targets signal. Used only for quantitative models. Default: `1`
 
-
 ### `--output_activation`
 
 The output activation to use for the model. No other options are considered in the publication. Test at your own risk. Default: `sigmoid`
 
 ### `--chroms`
 
-The list of chromosomes to limit the study to. These include the training and validation chromosomes. Default: ```["chr2", "chr3", "chr4", "chr5", "chr6", "chr7", "chr9", "chr10", "chr11", "chr12", "chr13", "chr14", "chr15", "chr16", "chr17", "chr18", "chr19", "chr20", "chr21", "chr22", "chrX"]```
+The list of chromosomes to limit the study to. These include the training and validation chromosomes. Default: ```["chr2", "chr3", "chr4", "chr5", "chr6", "chr7", "chr9", "chr10", "chr11", "chr12", "chr13", "chr14", "chr15", "chr16", "chr17", "chr18", "chr19", "chr20", "chr21", "chr22", "chrX"]```.
 
 ### `--tchroms`
 
-The list of chromosomes to use for training only. Default: ```["chr3", "chr4", "chr5", "chr6", "chr7", "chr9", "chr10", "chr11", "chr12", "chr13", "chr14", "chr15", "chr16", "chr17", "chr18", "chr20", "chr21", "chr22"]```
+The list of chromosomes to use for training only. Default: ```["chr3", "chr4", "chr5", "chr6", "chr7", "chr9", "chr10", "chr11", "chr12", "chr13", "chr14", "chr15", "chr16", "chr17", "chr18", "chr20", "chr21", "chr22"]```.
 
 ### `--vchroms`
 
-The list of chromosomes to use for validation only. Default: ```["chr2", "chr19"]```
+The list of chromosomes to use for validation only. Default: ```["chr2", "chr19"]```.
 
 ### `--arch`
 
-The architecture to use for the neural network. Default: `DCNN_V2`
+The architecture to use for the neural network. maxATAC-v1.0.6 supported architectures include `DCNN_V2, RES_DCNN_V2, MM_DCNN_V2 and MM_Res_DCNN_V2`. Default: `DCNN_V2`.
```

Comment:
We dont want to include all these types of architectures as params since they are no longer supported in the release of maxatac. The listed options were tested prior to release.

Reply:
Ooh, okay! I based these off of what is supported by the PyPi release of maxATAC-v1.0.6, but I can delete this. Thanks for the feedback!